app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditActivityPresenter.kt

```diff
@@ -121,6 +126,12 @@ class ProfileEditActivityPresenter @Inject constructor(
               }
             }
           )
-      }.create().show()
+      }.create()
+    dialog.show()
+  }
```

Comment:
can you use scope function [apply](https://kotlinlang.org/docs/scope-functions.html#apply) here

Reply:
@FareesHussain We can,  but do we need to?
I think, with apply, code will be something like:
```
AlertDialog.Builder(..)
    .apply {
        dialog = create()
        dialog.show()
    }
```

Reply:
![image](https://user-images.githubusercontent.com/54740946/113087463-47abdc00-9201-11eb-89ad-90af89e48ebb.png)
This is what I meant here
or you can just change it to `.create().show()` instead of `dialog.show()`

Reply:
@FareesHussain If I am not wrong, in the code snippet you shared, you used the apply function on AlertDialog.Builder and so `dialog` will be initialized to an AlertDialog.Builder instead of an AlertDialog after performing all the below operations on it.

Reply:
Ah, I was a little confused. Yes, you are right.

You can resolve this conversation now.

Reply:
Okay